dev/io.openliberty.org.jboss.weld6/test/io/openliberty/weld/jsf/WeldJSFCheckSumTest.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.weld.jsf;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * These tests look for changes in the FacesUrlTransformer & ConversationAwareViewHandler classes.
+ * If the checksum changes, then the latest classes should investigated and ported over to com.ibm.ws.jsfContainer
+ * See issue https://github.com/OpenLiberty/open-liberty/issues/14524
+ */
+public class WeldJSFCheckSumTest {
+
+    File bundle;
+
+    //TODO: this will all need to be updated once the EE10 version of JSF has been done
+    String bundleFileName = "io.openliberty.org.jboss.weld5.jar";
```

Comment:
This should say weld6 ... but then maybe the previous version needs updating anyway, as suggested by the comment?